src/jit/simd.cpp

```diff
@@ -3127,18 +3127,18 @@ GenTree* Compiler::impSIMDIntrinsic(OPCODE                opcode,
             GenTree*             dupOp1    = fgInsertCommaFormTemp(&op1, op1Handle);
 
             // Widen the lower half and assign it to dstAddrLo.
-            simdTree = gtNewSIMDNode(simdType, op1, nullptr, SIMDIntrinsicWidenLo, baseType, size);
-            GenTree* loDest =
-                new (this, GT_BLK) GenTreeBlk(GT_BLK, simdType, dstAddrLo, getSIMDTypeSizeInBytes(clsHnd));
+            simdTree        = gtNewSIMDNode(simdType, op1, nullptr, SIMDIntrinsicWidenLo, baseType, size);
+            GenTree* loDest = new (this, GT_BLK)
+                GenTreeBlk(GT_BLK, simdType, dstAddrLo, typGetBlkLayout(getSIMDTypeSizeInBytes(clsHnd)));
```

Comment:
Just trying to preserve the existing `OBJ`/`BLK` distinction for now to avoid surprises. As I explained in the past I do think that it would be best to use mostly `OBJ` nodes and keep `BLK`/`DYN_BLK` only as a `cpblk` representation. And maybe go vice versa - if we find that `cpblk` is used to copy 2 variables having the same type then change it to `OBJ` so that `BLK` occurs only in exotic circumstances (e.g. struct reinterpretation).